src/shared/credentials/userCredentialsUtils.ts

```diff
@@ -72,6 +72,18 @@ export class UserCredentialsUtils {
             || path.join(SystemUtilities.getHomeDirectory(), '.aws', 'config')
     }
 
+    /**
+     * @description Determines if credentials directory exists
+     * If it doesn't, creates credentials directory at:
+     * path.join(SystemUtilities.getHomeDirectory(), '.aws')
+     */
+    public static async generateCredentialDirectoryIfNonexistent(): Promise<void> {
+        const filepath = path.join(SystemUtilities.getHomeDirectory(), '.aws')
```

Comment:
Can you use `getCredentialsFilename` to get the path?

Reply:
✔️